.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: terraform fmt
+        run: |
```

```suggestion
        working-directory: ./terraform/
        run: |
```

The `.terraform-version` file is in `terraform`, so if we `cd` into there.

---

I'm not sure it's necessary if we have a `.terraform-version` at the root